modules/websocket/wsl_server.cpp

```diff
@@ -237,6 +255,7 @@ void WSLServer::poll() {
 		}
 		peer->tcp = conn;
 		peer->time = OS::get_singleton()->get_ticks_msec();
+		peer->headers = _headers;
```

Comment:
Instead of adding the property to the peer info, can't we just pass them as a parameter of `do_handshake` (passed by reference)?